pkg/ncproxy/nodenetsvc/v1/doc.go

```diff
@@ -6,6 +6,6 @@
 //
 // The mock service is compiled using the following command:
 //
-// mockgen -source="nodenetsvc.pb.go" -package="nodenetsvc_mock" > mock\nodenetsvc_mock.pb.go
+// mockgen -source="nodenetsvc.pb.go" -package="nodenetsvc_v1_mock" > mock\nodenetsvc_mock.pb.go
```

Comment:
What is this coming from?

Reply:
where is the mockgen line coming from? That's just the command that was used to make the mocked version of the api. However, I do need to update this to use the go generate directive now that I've rebased on Hamza's recent PR https://github.com/microsoft/hcsshim/pull/1802. Same for the v0 doc file.

Reply:
I'm confused why this file is changing at all, since IIRC nothing in nodenetsvc/v1 should be changing in this PR.

Reply:
Oh shoot, looking at the wrong PR, you're right. This is left over. Removing.